gapis/api/vulkan/api/enums.api

```diff
@@ -1111,6 +1111,8 @@ enum VkAttachmentLoadOp: u32 {
 enum VkAttachmentStoreOp: u32 {
   VK_ATTACHMENT_STORE_OP_STORE     = 0x00000000,
   VK_ATTACHMENT_STORE_OP_DONT_CARE = 0x00000001,
+  // @extension("VK_QCOM_render_pass_store_ops")
+  VK_ATTACHMENT_STORE_OP_NONE      = 1000301000,
```

Comment:
Isn't it `VK_ATTACHMENT_STORE_OP_NONE_QCOM`?

Reply:
Yes but it's core now

Reply:
https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/VkAttachmentStoreOp.html